packages/zoe/test/offerArgsUsageContract.js

```diff
@@ -2,7 +2,9 @@
 const start = zcf => {
   /** @type {OfferHandler} */
   const handler = (_seat, offerArgs) => {
+    // @ts-expect-error xxx HandleOffer
```

Comment:
I'm a bit reluctant to speak for @Chris-Hibbert and @erights re maintaining zoe package types in typescript rather than JSDoc, but I suppose this sort of thing shows that maintenance was suffering anyway.

Reply:
This approach seems to get us closer to single-source for API docs (#4291). @erights said that made would make this well worth it.